libbeat/common/uuid.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+// GenerateUUID is used to generate a random UUID
+// This was copied from https://github.com/hashicorp/uuid/blob/master/uuid.go
```

Comment:
@tsg I will suggest a library that supports all UUID versions so it is also easy to switch.